tests/model_fixtures.py

```diff
@@ -1,27 +1,43 @@
 """
 This is where we put all the data about creating / updating models
 """
-from dataclasses import dataclass
-from typing import Dict, Sequence, Union
+from dataclasses import dataclass, field
+from typing import Dict, List, Sequence, Union
 
 APIOpts = Dict[str, Union[int, str]]
 
 
 @dataclass
-class ModelFixtures:
+class ModelFixture:
     path: str
     required_fields: Sequence[str]
     create_opts: Sequence[APIOpts]  # valid ways to create the model
     invalid_opts: Sequence[APIOpts]  # setting these on creation or update should error
     update_opts: Sequence[APIOpts]  # setting these on update should be valid
 
 
+@dataclass
+class ModelRelationFixture:
+    field_name: str  # Name of the field cooresponding to the related table
+    model_fixture: ModelFixture
+
+
+@dataclass
+class ModelFixtureWithDependencies(ModelFixture):
```

Comment:
Could we simplify things by just attaching this attribute to ModelFixture? An empty list (default) would mean no dependencies.

Reply:
Oh yeah! I'd initially dismissed this as impossible because of the circular type hinting. The missing link here was that I didn't realize (until just now) that `List["SomeClass"]` ForwardRef-style type hinting could make this possible. Change coming momentarily.

Reply:
✔️ https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/31/commits/ace3c1a02743c1e740d01906a4bd48a32099d87a